apps/backoffice-v2/src/pages/Entity/components/Case/hooks/useCaseActionsLogic/useCaseActionsLogic.tsx

```diff
@@ -66,6 +67,9 @@ export const useActions = ({ workflowId, fullName }: IUseActions) => {
       }),
     [mutateAssignWorkflow],
   );
+
+  const tag = useMemo(() => workflow?.tags?.find(t => tagToBadgeData[t]), [workflow?.tags]);
```

Comment:
can you use filter tags and append all badges that are being retrieved? 
only think about it for instance tagging "pending user input" 
it can be revision + pending user input.

( ask Nitzan OR review later )

Reply:
Nitzan says that he doesn't see this scenario happening right now